src/main/java/cam72cam/immersiverailroading/util/SwitchUtil.java

```diff
@@ -41,6 +41,18 @@ public static SwitchState getSwitchState(TileRail rail, Vec3d position) {
 			}
 		}
 
+		if (parent.info.switchForced != SwitchState.NONE) {
```

Comment:
Maybe use isSwitchstateForced here?

Reply:
Why not? I was actually considering removing that helper because it wasn't used, but that's a great use.